src/master/master_impl.cc

```diff
@@ -2090,36 +2078,14 @@ void MasterImpl::TabletNodeRecoveryCallback(std::string addr,
     }
     node->ResetQueryFailCount();
 
-    // recovery offline tablets
-    // unload should-not-exist tablets
+    // New tabletnode should not have any tablet.
     uint32_t meta_num = response->tabletmeta_list().meta_size();
-    for (uint32_t i = 0; i < meta_num; i++) {
```

Comment:
@caijieming-baidu  We have to assume the new ts has no tablet loaded, otherwise ‘multi-load’ would be inevitable.
This is the fundamental design of Tera.